### DISCUSSION TITLE
Add Runs-on and container image support to Defaults

### DISCUSSION THREAD

**Discussion by sf-jkollars** [community/community#17186] [Actions]
In a workflow with many jobs, instead of having to add a runs-on line for each

    jobs:
      job1:       
        runs-on: [self-hosted, dev]
      job2:
        runs-on: [self-hosted, dev]       
        

 have it as a defaults option
    
    defaults:
      runs-on: [self-hosted, dev]

**Comment by matmair** [comment]
That would be a cool feature
